conans/test/unittests/tools/files/test_downloads.py

```diff
@@ -157,6 +158,33 @@ def test_download_no_retries_errors(self, bottle_server):
         assert "Waiting" not in str(conanfile.output)
         assert "retry" not in str(conanfile.output)
 
+    def test_download_localfile(self):
+        conanfile = ConanFileMock()
+        conanfile._conan_requester = requests
+
+        file_location =  os.path.join(temp_folder(), "file.txt")
+        with open(file_location, 'w') as textfile:
+            textfile.write('this is some content\n')
```

Comment:
```suggestion
        save(file_location, "this is some content\n")
```

We have one line helpers for this.